src/plugins/explore/public/types.ts

```diff
@@ -154,7 +154,7 @@ export interface ExploreServices {
   overlays: CoreStart['overlays'];
 
   // From DataExplorerServices (since Explore incorporates DataExplorer functionality)
-  store: ReduxStore; // Redux store
+  store: AppStore; // Redux store
```

Comment:
This AppStore fixes type errors. ReduxStore is very generic and doe not know the context of our reducers and state